Source/Plugins/ImportExport/ImportExportFilters/NodesTrianglesToVtk.cpp

```diff
@@ -230,8 +222,7 @@ void NodesTrianglesToVtk::execute()
   {
 
     QString ss = QObject::tr(": Error creating Triangles VTK Visualization '%1'").arg(getOutputVtkFile());
-    setErrorCondition(-1);
-    notifyErrorMessage(getHumanLabel(), ss, -666);
+    setErrorCondition(-668, ss);
```

Comment:
This error code is wrong.

Reply:
See comment above.

Reply:
I wrote the original code and reviewed the current state of the code. I am ok with changing the -1 to -666